runtime/lua/vim/lsp/util.lua

```diff
@@ -672,6 +672,7 @@ do
       table.insert(cmd_parts, k.."="..v)
     end
     api.nvim_command(table.concat(cmd_parts, ' '))
+    api.nvim_command('highlight link ' .. highlight_name .. 'Sign ' .. highlight_name)
```

Comment:
should we use `default` here?

Reply:
I think ideally Neovim should use the LspDiagnostic's foreground and the SignColumn's background, but it doesn't matter much to me :).